inventory/server.lua

```diff
@@ -160,17 +160,24 @@ AddRemoteEvent("TransferInventory", function(player, item, amount, toplayer)
         CallRemoteEvent(player, "MakeNotification", _("not_enough_item"), "linear-gradient(to right, #ff5f6d, #ffc371)")
     else
         AddInventory(tonumber(toplayer), item, tonumber(amount))
-        RemoveInventory(tonumber(player ), item, tonumber(amount))
+        RemoveInventory(tonumber(player), item, tonumber(amount))
         
         CallRemoteEvent(player, "MakeNotification", _("successful_transfer", amount, item, GetPlayerName(tonumber(toplayer))), "linear-gradient(to right, #00b09b, #96c93d)")
         CallRemoteEvent(tonumber(toplayer), "MakeNotification", _("received_transfer", amount, item, GetPlayerName(player)), "linear-gradient(to right, #00b09b, #96c93d)")
     end
 end)
 
+AddRemoteEvent("RemoveFromInventory", function(player, item, amount)
+    if PlayerData[player].inventory[item] < tonumber(amount) then
+        CallRemoteEvent(player, "MakeNotification", _("not_enough_item"), "linear-gradient(to right, #ff5f6d, #ffc371)")
+    else
+        RemoveInventory(tonumber(player), item, tonumber(amount))
+    end
+end)
 
 function AddInventory(player, item, amount)
     local slotsAvailables = tonumber(GetPlayerMaxSlots(player)) - tonumber(GetPlayerUsedSlots(player))
-     if slotsAvailables >= amount or item == "cash"then
+     if slotsAvailables >= (amount * ItemsWeight[item]) or item == "cash" then
```

Comment:
Une erreur sur cette ligne lors de la vente : inventory/server.lua"]:180: attempt to perform arithmetic on a nil value (field '?')

Reply:
Il faut remette : 
> if slotsAvailables >= amount or item == "cash"then
pour regler le probleme